code/modules/qwerty/beecoin/beecoin.dm

```diff
@@ -5,16 +5,16 @@
 		if(M.stat != DEAD && !isbrain(M))
 			if(EMERGENCY_ESCAPED_OR_ENDGAMED)
 				if(!M.onCentCom() && !M.onSyndieBase())
-					inc_beecoin_count(BEECOIN_SURVIVE_REWARD)
+					inc_beecoin_count(BEECOIN_SURVIVE_REWARD, reason="Survived the shift.")
 				else
-					inc_beecoin_count(BEECOIN_ESCAPE_REWARD)
+					inc_beecoin_count(BEECOIN_ESCAPE_REWARD, reason="Survived the shift.")
```

Comment:
This should be "Escaped to centcomm" or something like that

Reply:
Next time do that as a code suggestion so I can just accept it instead of reopening my text editor.